### ISSUE TITLE
Level Progression

### ISSUE THREAD

**Issue by IrksomeZeus** [IrksomeZeus/CSSE220_FinalProject#24]
Should move automatically to the next level once all of the emeralds are collected. Maybe a small countdown before starting?

**Comment by IrksomeZeus** [comment]
Level automatically progresses. New issue created for the countdown feature and added to cycle 4 milestone.
